CHANGELOG.md

```diff
@@ -1,3 +1,49 @@
+# dbt_fivetran_log v2.3.0
+[PR #164](https://github.com/fivetran/dbt_fivetran_log/pull/164) includes the following updates:
+
+## Schema Data Changes
+**3 total changes • 3 possible breaking changes**
+> To prevent potential errors from naming and materialization updates, a `--full-refresh `**is required** after upgrading.
+
+| Data Models | Change Type | Old | New | Notes |
+| --- | --- | --- | --- | --- |
+| [`fivetran_platform__audit_table`](https://fivetran.github.io/dbt_fivetran_log/#!/model/model.fivetran_log.fivetran_platform__audit_table) | Model materialization (only for Databricks SQL Warehouse runtimes) | table | incremental | Added incremental model support for Databricks SQL Warehouses using the `merge` strategy. Previously, incremental support was limited to Databricks All Purpose Clusters via the `insert_overwrite` strategy. This update extends incremental functionality to SQL Warehouses, enabling more efficient model builds. |
+| [`fivetran_platform__audit_table`](https://fivetran.github.io/dbt_fivetran_log/#!/model/model.fivetran_log.fivetran_platform__audit_table) | New column | | `write_to_table_start_day` | Changed the column partitioned on from `sync_start_day` to the new `write_to_table_start_day`. The previous column could contain null values, which are not ideal for partitioning and may lead to unexpected behavior in incremental models. |
+| [`fivetran_platform__audit_table`](https://fivetran.github.io/dbt_fivetran_log/#!/model/model.fivetran_log.fivetran_platform__audit_table) | Deleted column | `sync_start_day` | | No longer in use given the above. |
+
+## Bug Fixes
+- Updated `fivetran_platform__mar_table_history` to include consumption records not associated with an active connection.
```

Comment:
We change the join on destination too, are we also including consumption records not associated with an active destination as well? Should we call that out? Or do we think connection is sufficient.